addons/mrp/models/product.py

```diff
@@ -22,11 +22,26 @@ class ProductTemplate(models.Model):
     produce_delay = fields.Float(
         'Manufacturing Lead Time', default=0.0,
         help="Average lead time in days to manufacture this product. In the case of multi-level BOM, the manufacturing lead times of the components will be added.")
+    is_kits = fields.Boolean(compute='_compute_is_kits', compute_sudo=False)
 
     def _compute_bom_count(self):
         for product in self:
             product.bom_count = self.env['mrp.bom'].search_count(['|', ('product_tmpl_id', '=', product.id), ('byproduct_ids.product_id.product_tmpl_id', '=', product.id)])
 
+    def _compute_is_kits(self):
+        domain = [('product_tmpl_id', 'in', self.ids), ('type', '=', 'phantom')]
+        bom_mapping = self.env['mrp.bom'].search_read(domain, ['product_tmpl_id'])
+        kits_ids = set(b['product_tmpl_id'][0] for b in bom_mapping)
+        for template in self:
+            template.is_kits = (template.id in kits_ids)
+
+    def _compute_show_qty_status_button(self):
+        super()._compute_show_qty_status_button()
+        for template in self:
+            if template.is_kits:
+                template.show_on_hand_qty_status_button = False
+                template.show_forecasted_qty_status_button = False
```

Comment:
Is there some cases where they are not available at the same time?

Reply:
yep, in enterprise odoo/enterprise@0bda231

Reply:
https://github.com/odoo/odoo/pull/75555#discussion_r700882290 :p

Reply:
I hate rental